packages/dds/test-dds-utils/package.json

```diff
@@ -92,22 +128,18 @@
 		"moment": "^2.21.0",
 		"prettier": "~3.0.3",
 		"rimraf": "^4.4.0",
-		"tsc-multi": "^1.1.0",
 		"typescript": "~5.1.6"
 	},
 	"fluidBuild": {
 		"tasks": {
 			"build:docs": {
 				"dependsOn": [
 					"...",
-					"api-extractor:commonjs"
+					"api-extractor:commonjs",
+					"api-extractor:esnext"
 				],
 				"script": false
-			},
-			"tsc": [
-				"...",
-				"@fluid-private/stochastic-test-utils#build:test"
-			]
```

Comment:
This should not have been removed. Yes, it is odd, but stochastic-test-utils builds test utils as part of build:test and test-dds-utils consumes those as part of product build.
We ought to edit fluid build tools to understand exports and which tasks produce them, etc.